operator/v1/types_csi_cluster_driver.go

```diff
@@ -279,8 +279,35 @@ type VSphereCSIDriverConfigSpec struct {
 	// If cluster Infrastructure object has a topology, values specified in
 	// Infrastructure object will be used and modifications to topologyCategories
 	// will be rejected.
+	// +listType=set
 	// +optional
 	TopologyCategories []string `json:"topologyCategories,omitempty"`
+
+	// globalMaxSnapshotsPerBlockVolume is a global configuration parameter that applies to volumes on all kinds of
+	// datastores. If unset it defaults to 3.
+	// Increasing number of snapshots above 3 can have negative impact on performance, for more details see: https://kb.vmware.com/s/article/1025279
+	// Volume snapshot documentation: https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-E0B41C69-7EEB-450F-A73D-5FD2FF39E891.html
+	// +kubebuilder:validation:Minimum=1
```

Comment:
how do I choose: "no snapshots"?

Reply:
Already discussed here: https://github.com/openshift/api/pull/1783#discussion_r1523271016

vmWare does not document this value but testing showed that it does not disable snapshots, but default is applied instead. I can add a clarifying statement:

`Setting this value to 0 does not disable volume snapshots, but results in default value being used.`

Reply:
> Already discussed here: [#1783 (comment)](https://github.com/openshift/api/pull/1783#discussion_r1523271016)

Please add a sentence in the documentation indicating that snapshots cannot be disabled.  On this item and the others if they cannot be disabled either.

Reply:
Added, for all 3 options as it's applicable to all of them.